.github/workflows/release.yml

```
@@ -0,0 +1,252 @@
+        working-directory: packages/js
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  # Wheels + PyPI (Python)
+  py-wheels-and-pypi:
+    if: ${{ needs.release-please.outputs.py_created == 'true' }}
+    needs: release-please
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix: { os: [ubuntu-latest, macos-latest, windows-latest] }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)